### DISCUSSION TITLE
MPD support

### DISCUSSION THREAD

**Discussion by itzmanish** [pear-devs/pear-desktop#970] [Ideas]
Do we have plan for integrating MPD client in youtube-music?

**Comment by Zo-Bro-23** [comment]
Feel free to create a PR to add the integration yourself! Instructions are available in the README (new folder in `./plugins` with `back.js`, and optionally `menu.js`). As for others, @Araxeus @MiepHD interested?

**Reply by MiepHD** [reply]
I don't know what MPD is...

**Reply by Zo-Bro-23** [reply]
> I don't know what MPD is...

It's Linux specific. I mentioned you because you're a top contributor; wanted to see if you knew anything about Linux. Do you know who did the MPRIS PR? Maybe they'll know something about MPD.

**Reply by Araxeus** [reply]
> Do you know who did the MPRIS PR?

https://github.com/th-ch/youtube-music/pulls?q=is%3Apr+mpris

you are talking to the creator of the MPRIS plugin 😂😂😂

**Reply by Zo-Bro-23** [reply]
> > Do you know who did the MPRIS PR?
> > https://github.com/th-ch/youtube-music/pulls?q=is%3Apr+mpris
> 
> you are talking to the creator of the MPRIS plugin joyjoyjoy

Right. Have any idea what MPD is?

**Reply by Araxeus** [reply]
I was talking about the creator of this discussion. @itzmanish 

He originally created the MPRIS plugin

I don't even have Linux, so no idea

**Reply by Zo-Bro-23** [reply]
Oh, ok. @itzmanish can you make a plugin for MPD?

**Reply by itzmanish** [reply]
Umm. I thought someone else might like to take this up. But no problem, I will do it xD.
